scipy-notebook/Dockerfile

```diff
@@ -60,6 +60,17 @@ RUN git clone https://github.com/PAIR-code/facets.git && \
 # Import matplotlib the first time to build the font cache.
 ENV XDG_CACHE_HOME="/home/${NB_USER}/.cache/"
 
+ARG CUSTOM_FONT=""
+RUN if [ "$CUSTOM_FONT" == "Source Han Sans CN" ]; then \
+       PYV=$(ls /opt/conda/lib | grep ^python); \
+       wget --quiet -P /opt/conda/lib/$PYV/site-packages/matplotlib/mpl-data/fonts/ttf/ https://mirrors.cloud.tencent.com/adobe-fonts/source-han-sans/SubsetOTF/CN/SourceHanSansCN-Normal.otf; \
```

Comment:
Creating a variable MPL_DATA `/opt/conda/lib/$PYV/site-packages/matplotlib/mpl-data/` will make the code easier to read.